review.css

```diff
@@ -0,0 +1,134 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: Arial, sans-serif;
+}
+
+body{
+    background: #fff;
+}
+
+.header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem;
+    background-color: #D8DBDE;
+}
+
+.search-bar{
+    flex-grow: 1;
+    display: flex;
+    margin: 0 2rem 0 4rem;
+
+}
+
+.search-input{
+    height: 40px;
+    width: 100%;
+    font-size: 17px;
+    padding: .4em;
+    margin-left: 150px;
+    border: 1px solid #777;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+}
+
+.search-btn {
+    margin-right: 300px;
+    display: flex;
+    align-items: center;
+    padding: 0 1.25rem;
+    border: 1px solid #777;
+    background-color: #f8f8f8;
+    border-left: none;
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    cursor: pointer;
+    height: auto;
+}
+
+.search{
+    background-color: #f0f0f0;
+    height: 38px;
+    width: 30px;
+}
+
+.mic{
+    margin-left: -310px;
+    height: 34px;
```

Comment:
I would suggest using position and float property instead of margin in negative value!